src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => {
+        const { title, completed } = todo;
+
+        return (
+          <div className={classNames('todo', { completed })}>
```

Comment:
add `key`

Reply:
Key and class have the same names, so linter recomented me use only key